### ISSUE TITLE
[Request] More administration functions

### ISSUE THREAD

**Issue by androidacy-user** [watzon/fenneco#1]
This userbot seems to be locking in things like bans, gbans, pins, etc

I mean sure I could use rose but what's the fun in that

**Comment by watzon** [comment]
Yep those are coming. I've been holding off because those need a database, but I'll be adding them soon.
